src/components/RichTextEditor.tsx

```diff
@@ -118,15 +124,21 @@ export default function RichTextEditor({
 
   return (
     <Styles focus={focus} disabled={disabled}>
-      <Editor
-        disabled={disabled}
-        init={{
-          skin: false,
-          content_css: false,
-          content_style: [
-            theme.palette.mode === "dark" ? contentCssDark : contentCss,
-            theme.palette.mode === "dark" ? contentUiCssDark : contentUiCss,
-            `
+      <Suspense fallback={<></>}>
+        {theme.palette.mode === "dark" ? (
+          <RichTextEditorDarkCSS />
+        ) : (
+          <RichTextEditorLightCSS />
+        )}
+        <Editor
```

Comment:
```suggestion
        <GlobalStyles
          styles={{
            ".tox": {
              "& .tox-menu.tox-menu, &.tox-tinymce-aux .tox-toolbar__overflow.tox-toolbar__overflow":
                {
                  backgroundColor: theme.palette.background.paper,
                  backgroundImage:
                    "linear-gradient(rgba(255, 255, 255, 0.12), rgba(255, 255, 255, 0.12))", // elevation 8
                  boxShadow: theme.shadows[8],
                  border: "none",
                  borderRadius: (theme.shape.borderRadius as number) * 2,
                },

              "& .tox-collection--list": {
                "& .tox-collection__group.tox-collection__group": {
                  padding: theme.spacing(0.5),
                },
                "& .tox-collection__item": {
                  borderRadius: theme.shape.borderRadius,
                },
              },

              "&.tox-tinymce-aux .tox-toolbar__overflow.tox-toolbar__overflow":
                {
                  padding: theme.spacing(0.5, 0),
                },
              "& .tox-tbtn.tox-tbtn": {
                borderRadius: theme.shape.borderRadius,
                margin: 0,
              },
            },
          }}
        />
        
        <Editor
```

Adding GlobalStyles here to style the dropdowns. These need to be global because they’re mounted to the DOM in a React Portal, so the `<Styles>` above (which only apply to children) won’t work here.